### ISSUE TITLE
Commas between CSS blocks should be invalid

### ISSUE THREAD

**Issue by tonytonyjan** [stylelint/stylelint#3498]
> Clearly describe the bug

Refers to the [CSS3 syntax doc](https://www.w3.org/TR/css-syntax-3), I do not think a comma can be put between CSS rules. I tried many kinds of configurations to detect this case but in vain.

> Which rule, if any, is the bug related to?

There has not been a rule for this yet, I guess.

> What CSS is needed to reproduce the bug?

To reproduce, You can copy and paste the stylesheet below to https://stylelint.io/demo/.

```css
.foo {
  color: red;
}

, /* COMMA HERE*/
.bar {
  color: green;
}

```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "extends": ["stylelint-config-standard"]
}
```

> Which version of stylelint are you using?

```
"stylelint": "^9.3.0",
"stylelint-config-standard": "^18.2.0",
```

> How are you running stylelint: CLI, PostCSS plugin, Node API?

First, save the stylesheet into `sample.css`, then execute `stylelint sample.css`.

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No

> What did you expect to happen?

Prints error messages like:

```
Unexpected token ',' at line 5, col 1.
```

> What actually happened (e.g. what warnings or errors did you get)?

Nothing happened, it passed.

**Comment by hudochenkov** [comment]
Additional information:

PostCSS [parses this](http://astexplorer.net/#/gist/99cc1cbe2169a2d5c487cd0a68975120/41188f85531e926f2617bcb456d199c7d87385c4) as a part of selector.

**Comment by evilebottnawi** [comment]
Also we don't lint invalid CSS

**Comment by jeddy3** [comment]
@tonytonyjan Thanks for the report and for using the template.

In [our VISION doc](https://github.com/stylelint/stylelint/blob/master/VISION.md#possible-errors) we say:

> Invalid code is best handled by emerging dedicated tools e.g. csstree - a language parser with syntax validation. 

You can use [the csstree validator](https://github.com/csstree/validator), alongside stylelint, if you want complete specification validation.

Alternatively, you can create a [stylelint plugin](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/plugins.md) to validate this specific bit of syntax. There was a rule in stylelint that was removed in `8.0` that did something similar. You could use [the code of that](https://github.com/stylelint/stylelint/pull/1977/files) as a foundation of your plugin.
